### PR TITLE
Update Postgres README.md

### PR DESCRIPTION
**Pull request by tiagograf-AOT**
## Status
**Ready**

## Fixes or Implements [AB#8465](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8465)
- [ ] Enhancement
- [ ] Bug
- [x] Documentation

## Description:
Adds restore backup documentation to postgres db.

## Testing
- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

### CODE REVIEW

File: Tools/Postgres/README.md [hunk 1]
```diff
@@ -63,3 +63,32 @@ The script below sets the leader to node 2:
...
+
+## Restore Backup
+
+To download the backup to your local computer:
+
+``` bash
+oc rsync <backup-pod-name>:/backups/daily/<date> <local-folder>
+```
+
+Extract backup using gzip:
```
Comment (sslaws) [inline Tools/Postgres/README.md:75]: What would we use this for?

File: Tools/Postgres/README.md [hunk 2]
```diff
@@ -63,3 +63,32 @@ The script below sets the leader to node 2:
...
+
+Connect to the master database pod using port-forward (See 'Connection to the Database').
+
+Manually create the database:
+
+``` bash
+psql -h localhost -p 5432 -U postgres -c 'create database gateway;'
+```
+
+Execute the backup script:
```
Comment (sslaws) [inline Tools/Postgres/README.md:89]: This is for Create the backup right?

File: Tools/Postgres/README.md (hunk 1 above)
Comment (tiagograf-AOT) [reply Tools/Postgres/README.md]: this copies the folder from the pod to the local folder.

File: Tools/Postgres/README.md (hunk 2 above)
Comment (tiagograf-AOT) [reply Tools/Postgres/README.md]: no this executes the script containing the structure + data.

**Pull request merged**